core/src/main/java/com/datastax/oss/driver/internal/core/metrics/DropwizardMetricsFactory.java

```diff
@@ -41,24 +46,42 @@
 public class DropwizardMetricsFactory implements MetricsFactory {
 
   private static final Logger LOG = LoggerFactory.getLogger(DropwizardMetricsFactory.class);
+  private static final Duration LOWEST_ACCEPTABLE_EVICTION_TIME = Duration.ofHours(1);
```

Comment:
That seems a bit aggressive. They are probably many applications where hundreds of requests per second is the norm, I can see how someone might want to go lower in those cases.

We want to prevent absurd values, like less than a few seconds. But we could go as low as 5 minutes.

Reply:
good idea, it is too strict now. Changed